deploy/template.yaml

```diff
@@ -2543,6 +2543,39 @@ Resources:
         SSEType: KMS
         KMSMasterKeyId: !GetAtt DynamoDBKmsKey.Arn
 
+  SessionCredentialsTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      # checkov:skip=CKV_AWS_28: Point in time recovery is not necessary for this table.
```

Comment:
This seems to contradict the PITR config below? I think I agree with the comment and we should remove PITR...

Reply:
We have this on the other tables - and I reasoned it in my head that in envs other than prod we don't enable it. Which is why the `checkov:skip` is neccesary, to allow the pipelines to pipeline in the lower envs. 
I guess that as this is such a short lived table if we ever actually needed it, it'd probably be pointless by the time we'd actually managed to use the functionality. Will remove.

Reply:
Yeah - e.g. it looks like the main session table doesn't have it